src/components/List/List.tsx

```diff
@@ -86,6 +90,15 @@ class List extends UIComponent<Extendable<IListProps>, any> {
   // List props that are passed to each individual Item props
   static itemProps = ['debug', 'selection', 'truncateContent', 'truncateHeader', 'variables']
 
+  private containerFocusHandler = new ContainerFocusHandler(
```

Comment:
This would be nice to have, and that's was the initial way how I did it, but there is a reason, why not to do it:
- FocusContainer/FocusableItem will have a dependency on React.Component, which I want to avoid, to keep the focusable logic framework agnostic

Reply:
we can have the framework agnostic, but still provide factory functions specific for react. for other frameworks, we can create different factory functions.